src/components/FileTree.vue

```
@@ -231,6 +236,26 @@ export default {
         revisionChanged(index) {
             this.$emit('revision', this.revisionOptions[index].value);
         },
+
+        revisedFileRoute(f) {
+            let fileId;
+
+            if (f.ids != null) {
+                [, fileId] = f.ids;
+            } else if (f.revision != null) {
+                fileId = f.revision.id;
+            } else {
+                return '';
```

This should raise an error.

---

Yep.